Source/Core/DolphinQt/SkylanderPortal/SkylanderPortalWindow.h

```diff
@@ -0,0 +1,68 @@
+// Copyright 2022 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <array>
+#include <optional>
+
+#include <QCheckBox>
+#include <QDialog>
+#include <QGroupBox>
+#include <QLineEdit>
```

Comment:
Some of these includes are unnecessary; for instance you can forward declare QLineEdit and only include QLineEdit in the .cpp file (like what you've done for QPushButton) to reduce build times.